contracts/LinniaPermissions.sol

```diff
@@ -43,43 +45,46 @@ contract LinniaPermissions is Ownable {
 
     /* External functions */
 
-    /// Give a viewer access to a record
-    /// @param fileHash the hash of the unencrypted file
-    /// @param viewer the user being allowed to view the file
-    /// @param ipfsHash the IPFS hash of the file encrypted to viewer
-    function grantAccess(bytes32 fileHash, address viewer, bytes32 ipfsHash)
+    /// Give a viewer access to a medical record owned by a patient
+    /// @param rootHash the root hash of the linnia record
+    /// @param viewer the user being permissioned to view the data
+    /// @param dataUri the ipfs path of the re-encrypted data
```

Comment:
is reference to ipfs necessary

Reply:
for now in the comment we say ipfs api otherwise it would be confusing

Reply:
👍